README.md

```diff
@@ -261,73 +259,350 @@ The SDK provides a simple interface for all the [media APIs mentioned here](http
+
+```
+
+**9. Delete File**
+
+Accepts the file ID and delete a file as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/delete-file).
+
+```java
+String fileId="your-file-id";
+ResponseMetaData result=ImageKit.deleteFile(fileId);
+
+```
+
+**10. Delete FileVersion**
+
+Accepts an object of class `DeleteFileVersionRequest` specifying the parameters to be used to delete file version. All parameters specified in the [documentation here](https://docs.imagekit.io/api-reference/media-api/delete-file-version) can be passed via their setter functions to get the results.
+
+```java
+DeleteFileVersionRequest deleteFileVersionRequest = new DeleteFileVersionRequest();
+deleteFileVersionRequest.FileId="629d95278482ba129fd17c97";
+deleteFileVersionRequest.VersionId="629d953ebd24e8ceca911a66";
+ResponseMetaData resultNoContent = ImageKit.deleteFileVersion(deleteFileVersionRequest);
+
+```
+
+**11. Delete files (bulk)**
+
+Accepts the file IDs to delete files as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/delete-files-bulk).
+
+```java
+List<String> fileIds = new ArrayList<>();
+fileIds.add("your-file-id");
+fileIds.add("your-file-id");
+fileIds.add("your-file-id");
+
+ResponseMetaData result=ImageKit.bulkDeleteFiles(fileIds);
+
+```
+
+**12. Copy file**
+
+Accepts an object of class `CopyFileRequest` specifying the parameters to be used to copy file. All parameters specified in the [documentation here](https://docs.imagekit.io/api-reference/media-api/copy-file) can be passed via their setter functions to get the results.
+
+```java
+CopyFileRequest copyFileRequest = new CopyFileRequest();
+copyFileRequest.SourceFilePath="/w2_image.png";
+copyFileRequest.DestinationPath="/Gallery/";
+copyFileRequest.IncludeFileVersions=true;
+ResultNoContent resultNoContent = ImageKit.copyFile(copyFileRequest);
+
+```
+
+**13. Move file**
+
+Accepts an object of class `MoveFileRequest` specifying the parameters to be used to move file. All parameters specified in the [documentation here](https://docs.imagekit.io/api-reference/media-api/move-file) can be passed via their setter functions to get the results.
+
+```java
+MoveFileRequest moveFileRequest = new MoveFileRequest();
+moveFileRequest.SourceFilePath="/Gallery/w2_image.png";
+moveFileRequest.DestinationPath="/";
+ResponseMetaData resultNoContent = ImageKit.moveFile(moveFileRequest);
+
+```
+
+**14. Rename file**
+
+Accepts an object of class `RenameFileRequest` specifying the parameters to be used to rename file. All parameters specified in the [documentation here](https://docs.imagekit.io/api-reference/media-api/rename-file) can be passed via their setter functions to get the results.
+
+```java
+RenameFileRequest renameFileRequest = new RenameFileRequest();
+renameFileRequest.FilePath="/w2_image.png";
+renameFileRequest.NewFileName="w2_image_s.png";
+renameFileRequest.setPurgeCache=true;
```

Comment:
`renameFileRequest.setPurgeCache=true;`, this doesn't seems like a setter function being called. It's more like a simple value is being set. What am I missing?